.github/workflows/build_rust_docs.yaml

```diff
@@ -6,6 +6,11 @@ on:
   pull_request:
     branches: [main]
 
+# See https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-using-concurrency-to-cancel-any-in-progress-job-or-run
+concurrency:
+  group: build-rust-docs-${{ github.head_ref || github.ref }}
```

Comment:
Is there a way of parametrizing this with the actual workflow name, so we can copy paste it everywhere?

Reply:
Good idea, I think `env.GITHUB_WORKFLOW` might work. I'll try.